README.md

```diff
@@ -690,6 +690,18 @@ public class Example {
 }
 ```
 
+Default `QeuryMapEncoder` use `Field` property and it does not support field that extend from super class. We recommend that you use `BeanQueryMapEncoder` who will be generated using java beans accessible getter property as query parameter names.
```

Comment:
typo in `QueryMapEncoder`

Reply:
Also, grammar is odd and tense is mixed up.  I believe that you are going for the following:

> When annotating objects with `@QueryMap`, the default encoder uses reflection to inspect provided objects `Fields` to expand the objects values into a query string.  If you prefer that the query string be built using getter and setter methods, as defined in the Java Beans API, please use the `BeanQueryMapEncoder`

Reply:
thanks，my english is very poor...